Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/Block-Grid-Editor/index.md

```diff
@@ -160,15 +161,22 @@ To scale an Area, click and drag the scale-button in the bottom-right corner of
 
 ### Area configuration
 
+An area 
+
 ![Block Grid - Area Configuration](images/BlockGridEditor_AreasConfiguration.png)
 
-* **Alias** - The alias is printed by `GetBlockGridHTML()`. Use the alias to target the Element representing the area.
+* **Alias** - The alias is used to identify this Area.
+It begin printed by `GetBlockGridHTML()` And used as name for the Area slot in Custom Views.
```

Comment:
I'm not sure I understand this sentence. Could it be rewritten different to be more clear?

Reply:
The addition/change is that now the Alias is not only printed to be used for CSS selectors. But is also used for the slots of which the Area is injected into a Block View.
This is important if you not want to use the Directive: `<umb-block-grid-render-area-slots></umb-block-grid-render-area-slots>` but ant to write the slots on your own.  hen you need to understand that the name of the slots are the Alias. :-D

I hope that helps, though it is quite technical, and only relevant to few.